mllib/src/test/scala/org/apache/spark/ml/regression/LinearRegressionSuite.scala

```diff
@@ -460,6 +460,10 @@ class LinearRegressionSuite extends SparkFunSuite with MLlibTestSparkContext {
     // Training results for the model should be available
     assert(model.hasSummary)
 
+    // Schema should be a superset of the input dataset
+    assert(model.summary.predictions.schema.fieldNames.toSet ===
```

Comment:
This may break if we add other output columns.  Can you check for containment, instead of equality?